src/stlink-lib/usb.c

```diff
@@ -710,6 +710,7 @@ int _stlink_usb_read_mem32(stlink_t *sl, uint32_t addr, uint16_t len) {
 
     cmd[i++] = STLINK_DEBUG_COMMAND;
     cmd[i++] = STLINK_DEBUG_READMEM_32BIT;
+    DLOG("using address %#010x\n", addr);
```

Comment:
does that really belong to that commit ? some lowlevel methods could use some better traces, but this is maybe out of that pr scope

Reply:
Removed